### PR TITLE
Example workload for running multiple tests over the same base

### PR DESCRIPTION
**Pull request by NathanTP**
A pain point (brought up in the mailing list) is that if you want to run a bunch of variations of a test on the same base image, you'll end up with a million copies of the image. Instead of creating a bunch of different workloads, you can trick Marshal into modifying the same image over and over.

It's not pretty because of #134 and #144. Once those are resolved, this should get a lot cleaner. For now though, I think it's worth documenting the workaround.

This PR only provides an additional example, it does not modify marshal at all.

### CODE REVIEW

File: example-workloads/many-tests/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Supporting Many Tests On One Base
+In same cases, you would like to run many different tests over the same image.
+If you were to make N different workloads, one for each test, you'd end up with
+N coppies of the binary and image, along with the associated build time of
```
Comment (a0u) [inline example-workloads/many-tests/README.md:4]:
```suggestion
N copies of the binary and image, along with the associated build time of
```

File: example-workloads/many-tests/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+## Basic Strategy
+The basic strategy is as follows:
+* Create a single base workload that contains everything needed to run every test.
+* Create many copies of a test workload, one for each test you want to run:
+    * All test json's should have the same workload name (the "name" option).
+    * Each test json should be in its own subdirectory (otherwise FireMarshal
+      will try to load them all and complain that multiple workloads have the
+      same name)
+    * Each test workload should have it's own workdir (the subdirectory), it's
+      own 'run' or 'command' option, and optionally it's own testing directory.
```
Comment (a0u) [inline example-workloads/many-tests/README.md:17]:
```suggestion
    * Each test workload should have its own workdir (the subdirectory), its
      own 'run' or 'command' option, and optionally its own testing directory.
```

File: example-workloads/many-tests/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+FireMarshal just thinks you modified the workload json and rebuilds only the
+changed parts.
+
+## Running the Example
+To use this example, you can run the 'runTest.sh' script:
+
+    $ ./runTest.sh t0/t.json
+    $ ./runTest.sh t1/t.json
+
+This script coppies the test workload into the top-level directory and runs
```
Comment (a0u) [inline example-workloads/many-tests/README.md:33]:
```suggestion
This script copies the test workload into the top-level directory and runs
```

**Comment by NathanTP** [comment]
Fixed. Ready for re-approval.

**Pull request merged**